WordPress/gutenberg
# Image block: Fix flaky tests

## What?

The image tests have recently been [flagged as flaky](https://github.com/WordPress/gutenberg/pull/51305#issuecomment-1623674974). This PR attempts to address that.

## Why?

We want our tests run efficiently and reliably.

## How?

I believe this issue may be caused by inconsistent uploading behavior in the `beforeEach()`. I've moved the upload logic outside of that call and instead do it manually for most of the tests now.

## Testing Instructions




Unfortunately, I'm not sure it's possible to test this locally. You can try running `npm run test:e2e:playwright -- image.spec.js` to make sure the tests pass, but the flakiness appears to only be present on Github.

### Testing Instructions for Keyboard

N/A

## Screenshots or screencast 
N/A
